sdk/core/azure-core-test/inc/azure/core/test/test_base.hpp

```diff
@@ -30,6 +31,19 @@ namespace Azure { namespace Core { namespace Test {
   class TestBase : public ::testing::Test {
 
   private:
+    void PrepareOptions(Azure::Core::_internal::ClientOptions& options)
+    {
+      // Set up client options depending on the test-mode
+      if (m_testContext.IsPlaybackMode())
+      {
+        options.Transport.Transport = m_interceptor->GetPlaybackTransport();
+      }
+      else if (!m_testContext.IsLiveMode())
+      {
+        options.PerRetryPolicies.push_back(m_interceptor->GetRecordPolicy());
+      }
```

Comment:
What do we expect to happen in the else case when it is neither?

Reply:
The options are not updated/modified.  That would be the case of LIVE mode.  We do not change anything to init an sdk client.

Reply:
> The options are not updated/modified. That would be the case of LIVE mode. We do not change anything to init an sdk client.

Consider adding that as a comment in source ...

```C++
// else The options are not updated/modified. That would be the case of LIVE mode.
```